Grids AI Assistant: Add (Update) API Descriptions Part 3 (#8815)
DevExpress/devextreme-documentation: api-reference/40 Common Types/15 grids/FilterExpr/FilterExpr.md

---
id: FilterExpr
module: common/grids
export: FilterExpr
type: BasicFilterExpr | CombinedFilterExpr | NegatedFilterExpr
---
---
##### shortDescription
A DataGrid/TreeList filter expression.

---

> **FilterExpr** is a union of multiple types that changes based on the represented filter expression. Refer to the following topics for additional information:

Reviewer: ```suggestion
**FilterExpr** is a union of multiple types whose structure depends on the represented filter expression. For additional information, see the following topics:
```
